DESCRIPTION

```diff
@@ -7,7 +7,7 @@ Maintainer: Philip Leifeld <philip.leifeld@glasgow.ac.uk>
 Description: Converts coefficients, standard errors, significance stars, and goodness-of-fit statistics of statistical models into LaTeX tables or HTML tables/MS Word documents or to nicely formatted screen output for the R console for easy model comparison. A list of several models can be combined in a single table. The output is highly customizable. New model types can be easily implemented.
 URL: http://github.com/leifeld/texreg/
 BugReports: https://github.com/leifeld/texreg/issues
-Suggests: nlme, survival, network, ergm, lme4 (>= 1.0), btergm, Zelig (>= 5.0-16)
+Suggests: nlme, survival, network, ergm, lme4 (>= 1.0), btergm, broom, Zelig (>= 5.0-16)
```

Comment:
As of recently, CRAN encourages indicating the minimum version number that is supported in brackets. Perhaps just throw in the current `broom` version number?